docs/managedCodeGen.md

```diff
@@ -0,0 +1,29 @@
+# Code generation for managed react-native modules
+
+This is the feature is applicable to C# implemented modules or viewmanagers. It will make sure that the code needed to integrate with the react-native application is constructed at build time instead of at runtime.
```

Comment:
```suggestion
This feature is applicable to C# implemented modules or `ViewManager`s. Its objective is to ensure the code needed to integrate with the `react-native` application is constructed at build time instead of at runtime.
```